hw/dv/tools/vcs/cover.cfg

```diff
@@ -8,14 +8,21 @@
 +tree tb.dut
 -module pins_if     // DV construct.
 -module clk_rst_if  // DV construct.
--moduletree prim_alert_sender  // prim_alert_sender is verified in FPV.
+// Prim_alert/esc pairs are verified in FPV and DV testbenches.
+-moduletree prim_alert_sender
+-moduletree prim_alert_receiver
+-moduletree prim_esc_sender
```

Comment:
Add to Xcelium `cover,ccf` as well?

https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/xcelium/cover.ccf

Reply:
Thank you for the reminder. I will do that.